yeonjae/programmers-1/나머지가1이되는수찾기.js

```diff
@@ -0,0 +1,16 @@
+function solution(n) {
+  let arr = [];
+   
+  for (let i = 1; i < n; i++) {
+    if (n % i === 1)
+      arr.push(i);
+    }
+    
+  return Math.min(...arr);
+}
```

Comment:
```suggestion
function solution(n) {
  const arr = [];
   
  for (let i = 1; i < n; i++) {
    if (n % i === 1)
      arr.push(i);
    }
    
  return Math.min(...arr);
}
```
- 문제 잘 푸셨어요 🙆‍♂️ 그런데 윤우님 풀이가  더 효율적인 코드라 한번 참고하시면 좋을 것 같습니다 👍 그리고 배열은 push를 해서 내부 값을 추가하더라도 재할당하는게 아니기 때문에 const로 선언하시는게 좋습니다 ~

Reply:
리뷰 확인했습니다 🙆🏻‍♀️